README.md

```diff
@@ -29,14 +29,22 @@ gem 'aws-embedded-metrics-customink'
 ```
 ## Usage
 
-Simple configuration.
+Simple configuration:
 
 ```ruby
 Aws::Embedded::Metrics.configure do |c|
   c.namespace = 'MyApplication'
 end
 ```
 
+Using the `Logger` sink to write to a log file:
+
+```ruby
+Aws::Embedded::Metrics.configure do |c|
+  c.sink = Aws::Embedded::Metrics::Sinks::Logger.new(Rails.logger, level: :info)
```

Comment:
So the pattern around the Node lib for CloudWatch on EC2 (https://github.com/awslabs/aws-embedded-metrics-python#configuration) is `c.log_group_name` and `c.log_stream_name`. Given that and knowing this is different, what do you think of this interface?

```ruby
Aws::Embedded::Metrics.configure do |c|
  c.logger = Rails.logger
  c.logger_level = :info # This could be the default too.
  c.sink = :logger
end
```

I noticed this is very much how DHH likes to see things. Ex, when the attribute API was be written. Using symbols and objects vs having users construct object literals which might be private or refactored later. That way the sink getter or via a `lookup_sink` private helper looks for a symbol first. I'm fine either way.

Reply:
🤔 would we anticipate that other sinks might have different initialization options? I'm open to this but it means either the footprint of `attr_accessors` for the config class grows with the product of each supported init signature or we end up with some method_missing / hash type thing. No dog in this fight for me, lemme know what you prefer.

Reply:
I'd say just merge it.